### ISSUE TITLE
Changed behavior in newest Windows terminal from Microsoft store

### ISSUE THREAD

**Issue by joltar** [microsoft/terminal#8678]
I have  been using Window Terminal to open and use a Pengwin installation.  I just installed the latest version.  In the previous Terminal I could  open the bash (?) for Pengwin or any other Linux installation.  Now the only option is for Azure, which I haven't installed.  I know the developers on GitHub don't suffer fools gladly, but I am just attempting to learn a little about  Linux after being defeated at coding by Fortran on an IBM mainframe at UCB in early 60's. (Leading to a beneficialcareer change to Medicine)  TMI. I'm sure altering the settings file would work, If I were able to code in Javascript.  But apparently the new Terminal no longer scans for Linux on installation.  This may be intentional, but is an inconvenience, if not a bug.  And there is apparently no convenient workaround. Apologies if this is a foolish observation. Or posted in the wrong place.

# Environment

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]
Windows Terminal version (if applicable):

Any other software?
```

# Steps to reproduce

# Expected behavior

# Actual behavior

**Comment by DHowett** [comment]
Hey there,
What do you get when you run `wsl.exe -l -v` from Command Prompt?

**Comment by joltar** [comment]
Dustin,
I uninstalled and reinstalled Pengwin,  Same  for Windows terminal.  Now no problems at all. So now I do feel foolish. I really like Terminal, but I wish I could open a new window e.g. command prompt as administrator.  Sorry to bother you.  And Stay Safe. Thanks from John  And Happy New Year.  But if I can't ever update Win 10 1909 by the time Microsoft stops supporting it, I may have to buy very reluctantly, an Apple.  Thanks again

**Comment by zadjii-msft** [comment]
Glad this issue was resolved! /dup #7199 is tracking the underlying bug here, which is that _sometimes_, WSL hasn't started before the Terminal, so we can't reliably query the list of avaliable distros.
